### PR TITLE
Circularity kit update 24.08 adoption view fixed

### PR DESCRIPTION
**Pull request by LuisRickert**
### Added

- add new figures
- add license files for new figures

### Changed

- change strucutre of the contents. Same content chapters are in different order
- change wording

[Issue 703](https://github.com/eclipse-tractusx/sig-release/issues/703)
[REPLACES PR 925](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/925)
## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

**Review by stephanbcbauer — COMMENTED** [review]
@LuisRickert thanks for the PR. Can you please check the DEPENDENCIES file? It seems to be outdated. Since this is a PR with more than 1000 changes, I have to create an IP Ticket on the Eclipse side. 
⇾ I will do this and link it to the PR.

Furthermore, thanks for adding the licence files for each picture … right now, there are many companies named. Is it just a copy & paste error? Or did they all contribute here? Or was this done by you?

@jSchuetz88 FYI

**Comment by LuisRickert** [comment]
> @LuisRickert thanks for the PR. Can you please check the DEPENDENCIES file? It seems to be outdated. Since this is a PR with more than 1000 changes, I have to create an IP Ticket on the Eclipse side. ⇾ I will do this and link it to the PR.
> 
> Furthermore, thanks for adding the licence files for each picture … right now, there are many companies named. Is it just a copy & paste error? Or did they all contribute here? Or was this done by you?
> 
> @jSchuetz88 FYI

I'll check the dependency file. Still i'm confused that this check is failing since i did a rebase for this fixed PR so it is up to date (Yesterday evening) and we didn't do any changes to that file.

wrt. the license files, i just used the exsisting one, since i was. just the person who was tasked to create the PR.

**Comment by LuisRickert** [comment]
Hey @stephanbcbauer i did some investigation. 

The "DEPENDENCIES" file is generated by the "3rd Party dependency"-Job based on the package-lock.json file. My reasoning:

- if the file in my PR is identical to the file the main branch, which it is, the result of the check should be the same
- this means the result of the check should be the same, ergo if it fails on my PR the check should also fail on the mail branch
 - i did the check locally on my PR and it failed as expected
 - the check also failed on the Main branch 
 - *but* if i run the check via the github actions on my fork the check fails for PR branch but *not* for the main branch
 -> this result confuses me

are the any guidance on this or have i missed something ?
dash-license tool is 1.1.1-Snapshot

File: docs-kits/kits/Circularity_KIT/page_changelog.md [hunk 1]
```diff
@@ -5,9 +5,25 @@ description: "Circularity KIT"
...
+### Changed
+
+- change structure of the contents. Same content chapters are in different order
+- change wording
+
+### Removed
+
+- ./.
+
 ## [1.3.0] - 17.09.2024
```
Comment (jSchuetz88) [inline docs-kits/kits/Circularity_KIT/page_changelog.md:27]: I'm confused here about the order? Why is version 1.2.0 newer than version 1.3.0?

**Review by jSchuetz88 — CHANGES_REQUESTED** [review]
Looks good to me. My only remark refers to the changelog (see comment below)

**Comment by stephanbcbauer** [comment]
@jSchuetz88 [IP issue created](https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/17587)

**Comment by jSchuetz88** [comment]
> @jSchuetz88 [IP issue created](https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/17587)

And already approved.

**Comment by stephanbcbauer** [comment]
PR Release relevant ⇾ minor issues will be resolved within a bugfix branch
- changelog wrong
- licence information for SVG need to be updated, source is missing

overall PR checked by @ther3sa , @jSchuetz88 , @stephanbcbauer

**Pull request merged**